common/interpreter/gipsy_scheme/packages/package.h

```diff
@@ -0,0 +1,3091 @@
+unsigned char packages_package_scm[] = {
+  0x28, 0x64, 0x65, 0x66, 0x69, 0x6e, 0x65, 0x20, 0x28, 0x63, 0x61, 0x61,
```

Comment:
this file gets built automatically & should NOT be included in the repository

Reply:
thank you. as we discussed, this is part of the tiny scheme fix.